### ISSUE TITLE
Plugin throwing error because of warning from npm

### ISSUE THREAD

**Issue by roger-lau** [sonatype-nexus-community/vscode-iq-plugin#130]
Getting error (bottom right) from the plugin when there is a warning (top left) from `npm install`.
Is there anyway to bypass that? I would expect a warning would not break the plugin. 

<img width="1330" alt="Screenshot 2020-10-21 at 15 23 07" src="https://user-images.githubusercontent.com/22045072/96686669-659f2600-13b1-11eb-8a99-e5723de78f1f.png">

**Comment by roger-lau** [comment]
Sharing my `package.json` file to reproduce the issue.

```
{
    "name": "sampleproject",
    "version": "0.0.1",
    "description": "Test Project 1",
    "dependencies": {
        "request-promise-native": "1.0.9"
    }
}
```

**Comment by roger-lau** [comment]
Alright, I found it. It was because this component was deprecated, together with its dependencies. So npm will not download the dependencies on `npm install` command, but, if you include it directly in `package.json`, it will still download it.

Even though there is no error during `npm install` but the packages are incomplete. So the plugin cannot process it. 

Solved: either upgrade to the recommended component from npm https://www.npmjs.com/package/request-promise-native
or include `request@2.34` in the `package.json`. The plugin will work.
